flake.nix

```diff
@@ -44,7 +44,7 @@
             default = simd;
             simd = with pkgs; mkShell {
               buildInputs = [
-                go_1_20 # Use Go 1.20 version
+                go_1_21 # Use Go 1.21 version
```

Comment:
We are blocked on this.

Reply:
https://github.com/NixOS/nixpkgs/pull/246935

Reply:
anyway to unblock this or are we going to be stuck like this going forward for future releases? Anyway we can merge this for now since the rocksdb stuff isnt needed right now

Reply:
We can skip the build step and create an issue to re-enable it if that's what we want 👍🏾

Reply:
mainly thinking since this is a blocker for the rc as well. Lets disable it and create an issue

Reply:
Ok! will do